src/imagewidget/index.css

```diff
@@ -0,0 +1,9 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+.jp-ImageWidget {
+  padding: 5px;
+  object-fit: contain;
```

Comment:
Unfortunately, `object-fit`, despite being the nicest solution to this issue, isn't supported by IE, so it can't be our answer for this.

[<img width="584" alt="object-fit on MDN" src="https://cloud.githubusercontent.com/assets/159529/16644666/952750d8-4416-11e6-9771-5d4f1e64b27d.png">](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)

https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit